<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>721</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Minecraft动物检测系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="playBtn">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="playSlider">
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="playLabel">
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpenVideo"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>视频(&amp;M)</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="actionNextFrame"/>
    <addaction name="actionPrevFrame"/>
    <addaction name="separator"/>
    <addaction name="actionRefreshFrame"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_M"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="toolDockWidget">
   <property name="windowTitle">
    <string>工具</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-96</y>
          <width>187</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QGroupBox" name="detGroupBox">
           <property name="title">
            <string>目标检测</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>NMS阈值</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>置信度阈值</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="detNMSSpinBox">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.300000000000000</double>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QDoubleSpinBox" name="detConfSpinBox">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.300000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="detDeviceComboBox">
              <item>
               <property name="text">
                <string>cuda:0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CPU</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>设备</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="imgProcGroupBox">
           <property name="title">
            <string>图像处理</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QGroupBox" name="histEqGroupBox">
              <property name="title">
               <string>直方图均衡化CLAHE</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="2" column="1">
                <widget class="QSpinBox" name="histLimitSpinBox">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="value">
                  <number>4</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Limit</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>TileSize</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QSpinBox" name="tileSizeSpinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>8</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="grayCheckBox">
              <property name="text">
               <string>灰度化</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="showHistogramCheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>显示直方图</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>视频播放</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="1">
             <widget class="QLabel" name="realFPSLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>实际FPS:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>缩放比例</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>FPS</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="scaleSpinBox">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="fpsSpinBox">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
              <property name="value">
               <double>30.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QPushButton" name="refreshFrameBtn">
              <property name="text">
               <string>刷新当前帧</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="classFilterDockWidget">
   <property name="windowTitle">
    <string>类别筛选</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="1">
      <widget class="QPushButton" name="clsMaskUncheckAllBtn">
       <property name="text">
        <string>全不选</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="clsMaskCheckAllBtn">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="clsMaskInverseCheckBtn">
       <property name="text">
        <string>反选</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QListView" name="classMaskListView">
       <property name="movement">
        <enum>QListView::Snap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>日志</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="outputTextBroser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenVideo">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>播放(&amp;P)</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停(&amp;U)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)...</string>
   </property>
  </action>
  <action name="actionNextFrame">
   <property name="text">
    <string>下一帧(&amp;N)</string>
   </property>
  </action>
  <action name="actionPrevFrame">
   <property name="text">
    <string>上一帧(&amp;B)</string>
   </property>
  </action>
  <action name="actionShowLogWnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>日志窗口</string>
   </property>
  </action>
  <action name="actionShowToolWnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具窗口</string>
   </property>
  </action>
  <action name="actionShowClassFilterWnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>类别筛选窗口</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="text">
    <string>暗色主题</string>
   </property>
  </action>
  <action name="actionLightTheme">
   <property name="text">
    <string>亮色主题</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRefreshFrame">
   <property name="text">
    <string>刷新当前帧</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
